### PR TITLE
Feature/helpful

### PR DESCRIPTION
**Pull request by cdcarter**
_aside: it seems like `black` changed spacing under class name, @davisagli can we get a lint check in CI?_

# Critical Changes

# Changes
- Print a message about the `info` commands after running task list or flow list.
- Do not print a full Python traceback when the CUMULUSCI_KEY Environment Variable isn't set, and use a more appropriate exception type in the keychain code.

# Issues Closed

### CODE REVIEW

File: cumulusci/cli/tests/test_config.py [hunk 1]
```diff
@@ -56,6 +58,12 @@ def test_load_project_config_error(self):
         with self.assertRaises(click.UsageError):
             config._load_project_config()
 
+    def test_load_keychain__no_key(self):
+        os.environ["CUMULUSCI_KEY"] = ""
+        with self.assertRaises(click.UsageError):
+            config = CliConfig()
+        os.environ["CUMULUSCI_KEY"] = self.key
```
[the remark this answers is not in the document]
Comment (cdcarter) [reply cumulusci/cli/tests/test_config.py]: nice!

File: cumulusci/cli/config.py [hunk 2]
```diff
@@ -121,14 +128,6 @@ def check_org_overwrite(self, org_name):
             pass
         return True
 
-    def check_keychain(self):
-        self.check_project_config()
```
[the remark this answers is not in the document]
Comment (cdcarter) [reply cumulusci/cli/config.py]: based on my read on _load_project_config, we'll have thrown a NotInProject error already from trying to init CliConfig

[the remark this answers is not in the document]
Comment (davisagli) [reply cumulusci/cli/config.py]: Ah, right. In that case we should probably move the helpful note about "cci project init" to the spot where ProjectConfigNotFound is raised, and remove check_project_config from here altogether?

File: cumulusci/cli/tests/test_config.py [hunk 3]
```diff
@@ -47,7 +47,10 @@ def test_load_project_config_no_file(self):
         config.global_config.get_project_config = mock.Mock(
             side_effect=ProjectConfigNotFound
         )
-        config._load_project_config()
+        try:
+            config._load_project_config()
+        except click.UsageError:
+            pass
         self.assertIsNone(config.project_config)
```
Comment (davisagli) [inline cumulusci/cli/tests/test_config.py]: Wouldn't it make more sense to just assert that this raises the exception?

**Pull request merged**